src/VisualStudio/Core/Def/Implementation/ChangeSignature/AddParameterDialog.xaml

```diff
@@ -32,13 +34,96 @@
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
```

Comment:
nit: I believe if they're all Auto you could just use a StackPanel.

Reply:
No need to change now though.

Reply:
Good catch -- the type field is supposed to expand to fill the space when the dialog is resized. Oops.

Reply:
Fixed.